samples/sqltransport/multi-instance/SqlServer_3/Receiver/Program.cs

```diff
@@ -38,4 +39,22 @@ await endpointInstance.Stop()
             .ConfigureAwait(false);
     }
 
+    static class ConnectionProvider
+    {
+        const string DefaultConnectionString = @"Data Source=.\SqlExpress;Database=ReceiverCatalog;Integrated Security=True";
+        const string SenderConnectionString = @"Data Source=.\SqlExpress;Database=SenderCatalog;Integrated Security=True";
+
+        public static async Task<SqlConnection> GetConnection(string transportAddress)
+        {
+            var connectionString = transportAddress.StartsWith("Samples.SqlServer.MultiInstanceSender")
```

Comment:
I think this is ok. The receiver endpoint will return default connection string in all cases, except when message is targeted to any destination that starts with `Samples.SqlServer.MultiInstanceSender` (which is the remote endpoint transport address)